### PR TITLE
Add navbar and FAB

### PR DESCRIPTION
**Pull request by mierzvoj**
Add iconized FAB

Jetpack Compose use in App design

### CODE REVIEW

File: app/src/main/res/drawable/ic_add_donation.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#8CB4B0"
```
Comment (matgrube) [inline app/src/main/res/drawable/ic_add_donation.xml]: Are those files still necessary? Please, make sure and delete if they are not :)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">Blooddonor-android</string>
+    <string name="description" />
```
Comment (matgrube) [inline app/src/main/res/values/strings.xml]: What's the purpose of this line?

**Review by matgrube — CHANGES_REQUESTED** [review]
Overall looks good, please check two comments and adjust if needed. Thanks!

File: app/src/main/res/drawable/ic_add_donation.xml (hunk 1 above)
Comment (mierzvoj) [reply app/src/main/res/drawable/ic_add_donation.xml]: No, they are not necessary anymore. To be removed.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (mierzvoj) [reply app/src/main/res/values/strings.xml]: This is unnecessary code. To be removed.

**Comment by mierzvoj** [comment]
Wait with merge, upper menu still pending.

**Review by matgrube — DISMISSED** [review]

**Review by matgrube — APPROVED** [review]

**Pull request merged**